app/queries/my_facilities/blood_pressure_control_query.rb

```diff
@@ -22,6 +20,23 @@ def initialize(facilities: Facility.all, cohort_period: {})
     @facilities = facilities
   end
 
+  def cohort_registrations_per_facility
```

Comment:
Can we rename this? This isn't the count of registrations anymore, so calling it `registrations_per_facility` is confusing.

Reply:
Addressed in 170a095.